src/Nancy.Testing/Browser.cs

```diff
@@ -54,9 +52,10 @@ public Browser(INancyBootstrapper bootstrapper, Action<BrowserContext> defaults
         /// <param name="path">The path that is being requested.</param>
         /// <param name="browserContext">An closure for providing browser context for the request.</param>
         /// <returns>An <see cref="BrowserResponse"/> instance of the executed request.</returns>
+        [Obsolete("Use async overload instead.", false)]

```

Comment:
See @grumpydev 's comment regarding forcing people to rewrite / fix all their tests. `True` here would make such happen too.